gap_quantization/models/squeezenet1_1.py

```diff
@@ -16,7 +16,7 @@
 }
 
 
-def conv2d(input_channels, out_channels, convbn=False, **kwargs):
+def conv2d(input_channels, out_channels, convbn, **kwargs):
```

Comment:
Why the default argument was removed?

Reply:
I guess it's a mistake, I've fixed it in next commit